app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -158,7 +158,7 @@ export async function isChainSupported(chainId: Hex): Promise<boolean> {
       `Error fetching supported chains from security alerts API`,
     );
   }
-  return supportedChainIds.includes(chainId);
+  return supportedChainIds.includes(chainId as Hex);
```

Comment:
Why was this cast necessary? It looks like it already has the type `Hex`

Reply:
It is not. I think this is just due to doing so many refactors and merges. I can fix it here, or fix it in a follow up PR.